### PR TITLE
gitignored anchor.toml and yarn.lock

### PR DESCRIPTION
**Pull request by mesozoic-technology**

### CODE REVIEW

File: Anchor.example.toml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+[scripts]
+test = "yarn run ts-mocha -p ./tsconfig.json -t 1000000 tests/**/*.ts"
+
+[test]
+startup_wait = 20000
+shutdown_wait = 2000
+upgradeable = false
+
+[[test.genesis]]
```
Comment (ith-harvey) [inline Anchor.example.toml]: can remove through `25 - 27` as we don't use metaplex

File: Anchor.toml [hunk 2]
```diff
@@ -12,10 +12,10 @@ url = "https://api.apr.dev"
```
Comment (ith-harvey) [inline Anchor.toml:12]: Since anchor.toml is in `.gitignore` should we delete it in this PR

**Review by ith-harvey — COMMENTED** [review]
Couple things

File: Anchor.toml (hunk 2 above)
Comment (mesozoic-technology) [reply Anchor.toml:12]: hmm I added it because it was adding my Anchor.toml changes to my commits

Comment (mesozoic-technology) [reply Anchor.toml:12]: Yeah good point

**Pull request merged**